### PR TITLE
You Must Become Strong

### PR DESCRIPTION
**Pull request by dsmith328**
## About The Pull Request
_slaps machine_ Do you love your City? You Must Become Strong for your City!

This baby is an implied aberration of "You Must Be Happy" and yet acts _very_ differently. Typically a good stat-trainer, this Abnormality is _fairly_ chill. When it hits 0 Qliphoth is spawns 3 boys that roam around and are kind of pain? They can self-ramp and need to be killed 3 times (or once with something like Smile!). You can also shove body parts into it for positive effects! Not possessing it's own EGO Gift, it turns your body parts themselves into EGO Gifts! Go on! Stick your ARM in there! Do it! You Must Become Strong!

The EGO of this abnormality serves two purposes:
The weapon is a gimmick weapon that is kinda cool but overall not great.
The armor is also kind of a gimmick, boasting a 5/-2/1/3 stat-line, providing really good resistances against the normal things that hurt your body and NOTHING ELSE.

This baby HAS Minion AND EGO Sprites thanks to `@Mel Táculo#1392` and `@Ubage#1029` on the discord, thank you!

## Why It's Good For The Game
It's a fuckin cool abno. Shove your hand in.
Fills up more of the stuff we have.
It has a few cool mechanics and, if you're interested in the EGO, provides a user for spare limbs.

### CODE REVIEW

File: code/modules/paperwork/records/info/he.dm [hunk 1]
```diff
@@ -618,3 +618,37 @@
...
+/obj/item/paper/fluff/info/he/you_strong
+	name = "T-09-140"
+	info = {"<h1><center>T-09-140</center></h1>    <br>
+	Name : You Must Become Strong    <br>
+	Risk Class : HE    <br>
+	Max PE Boxes : 16    <br>
+	Qliphoth Counter : 3    <br>
+	Work Damage Type : Red	<br>
+	Work Damage : Moderate	<br>
+	- When Repression Work was performed, the Qliphother Counter decreased.	<br>
```
Comment (SpaceJohn413) [inline code/modules/paperwork/records/info/he.dm:633]: qliphother

File: code/modules/paperwork/records/info/he.dm [hunk 2]
```diff
@@ -618,3 +618,37 @@
...
+	- After 3 works, the employee was prompted: 'Do you love your City?'<br>
+	- Answering 'Yes' heavily reduced the work success rate of the next work.	<br>
+	- Answering 'No' reduced the Qliphoth Countered by one.	<br>
```
Comment (SpaceJohn413) [inline code/modules/paperwork/records/info/he.dm:636]: qliphoth countered

File: code/datums/abnormality/_ego_datum/he.dm [hunk 3]
```diff
@@ -220,3 +220,12 @@
 /datum/ego_datum/weapon/songmini
 	item_path = /obj/item/gun/ego_gun/pistol/songmini
 	cost = 35
+
+//You Must Become Strong - GET POWERFUL
```
Comment (EgorDinamit) [inline code/datums/abnormality/_ego_datum/he.dm:224]: Abno name - EGO name
```suggestion
//You Must Become Strong - Get Strong
```

Comment (Coxswain-Navigator) [reply code/datums/abnormality/_ego_datum/he.dm:224]: Could also just call it screwloose - the name of the E.G.O. in limbus

File: code/modules/clothing/suits/ego_gear/he.dm [hunk 4]
```diff
@@ -265,3 +265,23 @@
 	attribute_requirements = list(
 							PRUDENCE_ATTRIBUTE = 40
 							)
+
+/obj/item/clothing/suit/armor/ego_gear/get_strong
+	name = "Get Strong"
+	desc = "It crunches your insides as you move... Do you love the City you live in?"
+	icon_state = "song"
+	armor = list(RED_DAMAGE = 50, WHITE_DAMAGE = -20, BLACK_DAMAGE = -20, PALE_DAMAGE = 50) // 60
```
Comment (EgorDinamit) [inline code/modules/clothing/suits/ego_gear/he.dm:273]: Another random pale armor? Just remove -20 armor.

**Review by Kitsunemitsu — APPROVED** [review]
Code looks fine, wasn't any issues with testing.
LGTM

**Review by Kitsunemitsu — APPROVED** [review]
Approving, had merge issues

**Pull request merged**